### ISSUE TITLE
npm install no longer supported for yaml-language-server

### ISSUE THREAD

**Issue by oeuftete** [mattn/vim-lsp-settings#446]
Looks like maybe https://github.com/redhat-developer/yaml-language-server/pull/458 was the upstream change that led to this?

Edit: I originally thought this was preventing the language server from running, but it does not.

```
Wrote to /home/me/.local/share/vim-lsp-settings/servers/yaml-language-server/package.json:

{
  "name": "yaml-language-server",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}


npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'yaml-language-server@0.22.0',
npm WARN EBADENGINE   required: {
npm WARN EBADENGINE     node: '*',
npm WARN EBADENGINE     npm: '\n\nERROR: Please use yarn, npm is not supported in this repository!!!\n\n',
npm WARN EBADENGINE     yarn: '^1.22.5'
npm WARN EBADENGINE   },
npm WARN EBADENGINE   current: { node: 'v16.5.0', npm: '7.19.1' }
npm WARN EBADENGINE }

added 24 packages, and audited 25 packages in 650ms

found 0 vulnerabilities

[Process exited 0]
```

**Comment by lunarxlark** [comment]
#450

**Comment by mattn** [comment]
Closed by #450
